ansible/condor.yml

```diff
@@ -45,58 +45,72 @@
         url: https://get.htcondor.org
         dest: /root/condor_install.sh
         mode: "0755"
-
-    - ansible.builtin.user:
-        user: clauncher
-        create_home: yes
-        append: true
-        groups: docker
```

Comment:
I moved this into the role for installing `condor-launcher`. It seemed more appropriate there.

Reply:
Update: I created a new role instead so that we can be sure that the job user exists on the submit node and all worker nodes.